backend/blueprints/spa_api/spa_api.py

```diff
@@ -61,10 +62,9 @@ def better_jsonify(response: object):
 ### GLOBAL
 
 @bp.route('/global/replay_count')
-def api_get_replay_count():
-    s = current_app.config['db']()
-    count = s.query(Game.hash).count()
-    s.close()
+@with_session
```

Comment:
Server won't run when using the same decorator on multiple endpoints unless you do `functools.wraps(decorated_function)` in the decorator.